content/news/2024/07/15/role-adverts.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Support role poisitons in the PyBaMM team
+date: 2024-07-15
+summary: We are advertising three positions for support roles within the PyBaMM team: communications lead, community manager and fundraising lead.
```

Comment:
```suggestion
summary: "We are advertising three positions for support roles within the PyBaMM team: communications lead, community manager and fundraising lead."
```
The error "failed to unmarshal YAML: yaml: line 3: mapping values are not allowed in this context" should get fixed with this suggestion.